src/constants/SocialLinks.tsx

```diff
@@ -0,0 +1,8 @@
+const SocialLinks = {
```

Comment:
Generally constant names in js are using upper snake case
```suggestion
export const SOCIAL_LINKS = {
```